src/components/navigation-page/navigation-page.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
```

Comment:
This navigation scheme should be used for each layout type now.

Reply:
Rather than create a new component, can we replace NavigationAccordion? We have no use for NavigationAccordion otherwise.